<launch>
	<include
		file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"
	/>
	<group>
		<remap from='joy' to='joyDrive'/>
		<node pkg='joy' name='joy0' type='joy_node'>
			<param name='dev' type='string' value='/dev/input/js0'/>
		</node>
	</group>
	<group>
		<remap from='joy' to='joyArm'/>
		<node pkg='joy' name='joy2' type='joy_node'>
			<param name='dev' type='string' value='/dev/input/js2'/>
		</node>
	</group>
</launch>